.jshintignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
I'm not sure what you mean here.  The node_modules directory is being properly ignored when I run jslint.  Why do I need a white-space?